.gitignore

```diff
@@ -42,3 +42,8 @@ Thumbs.db
 
 # compiled binaries
 /explorer.*
+/dist/vendor.181584baefd2a47c5eda.bundle.js
```

Comment:
@stdevDcmorales you should add a generic rule here e.g. `dist/*` or `dist/*.bundle.*` .